### PR TITLE
Resolves #2045

### LINKED ISSUE #2045: Clarify usage of the TaskPath Parameter
Module:  scheduledtasks

My expectation was that i can pass a TaskPath without any slash. 'ThePathName'
Got an Error

So I tried an TaskPath with an Slash in front '\ThePathName'
Got an Error

i never, never thought that i have to provide a slash in Front and in back. '\ThePathName\\'

Even the Examples are not clear because they use an Star ...  '\ThePathName\\*'

so please clarify usage of the TaskPath Parameter

### PR DESCRIPTION
**Pull request by velkovb**
Fix TaskPath parameter description
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2045

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/scheduledtasks/Disable-ScheduledTask.md [hunk 1]
```diff
@@ -129,14 +129,14 @@ Accept wildcard characters: False
 ```
 
 ### -TaskPath
-Specifies the path for a scheduled task in Task Scheduler namespace.
-You can use **\** for the root folder.
+Specifies an array of one or more paths for scheduled tasks in Task Scheduler namespace. You can use **"*"** for a wildcard character query.
+You can use **\\*** for the root folder. To specify a full TaskPath you need to include the leading and trailing **\**.
```
Comment (dariomws) [inline docset/windows/scheduledtasks/Disable-ScheduledTask.md:133]:
```suggestion
You can use **\\*** for the root folder. To specify a full TaskPath you need to include the leading and trailing **\\**.
```
I think you should fix this in all the cmdlets.

Comment (velkovb) [reply docset/windows/scheduledtasks/Disable-ScheduledTask.md:133]: Thanks, missed the escaping \ :(

File: docset/windows/scheduledtasks/Get-ScheduledTask.md [hunk 2]
```diff
@@ -127,13 +141,13 @@ Accept wildcard characters: True
 
 ### -ThrottleLimit
 Specifies the maximum number of concurrent operations that can be established to run the cmdlet.
-If this parameter is omitted or a value of `0` is entered, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer. 
+If this parameter is omitted or a value of `0` is entered, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```
Comment (JohanFreelancer9) [inline docset/windows/scheduledtasks/Get-ScheduledTask.md:144]:
```suggestion
If this parameter is omitted or a value of `0` is entered, then Windows PowerShell calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```

Comment (JohanFreelancer9) [reply docset/windows/scheduledtasks/Get-ScheduledTask.md:144]: Please remove the trademark symbol.

**Pull request merged**